PlayerController.cs

```diff
@@ -24,13 +30,15 @@ void Update()
     [Tooltip("How far player moves horizontally")][SerializeField] float xRange = 10f;
     [Tooltip("How far player moves vertically")][SerializeField] float yRange = 10f;
 
+    //enable when this script is enabled
```

Comment:
The `Enable()` and `Disable()` functions allow you to use Unity's new input management system!